mavros_extras/src/plugins/mag_calibration_status.cpp

```diff
@@ -99,6 +99,10 @@ class MagCalStatusPlugin : public plugin::Plugin
     mavlink::common::msg::MAG_CAL_REPORT & mr,
     plugin::filter::SystemAndOk filter [[maybe_unused]])
   {
+    if (mr.comapss_id >= sizeof(calibration_show) {
```

Comment:
Please fix misprint and use [`size() method`](https://en.cppreference.com/w/cpp/container/array/size).

Reply:
I just fixed it
Can you check it again?

Reply:
https://github.com/mavlink/mavros/runs/4485554446?check_suite_focus=true#step:4:548

Please fix CI.